scripts/port_passes_tests_to_lit.py

```diff
@@ -59,6 +51,15 @@ def port_test(args, test):
         with open(passes_file) as f:
             joined_passes = f.read().strip()
 
+    if 'translate-to-fuzz' in joined_passes or 'dwarf' in joined_passes:
+        warn('Skipping due to Windows issues')
+        return
+
+    for bad in ['noprint', 'metrics', 'fuzz', 'print', 'emit', 'dump']:
```

Comment:
```suggestion
    for bad in ['noprint', 'metrics', 'fuzz-exec', '-fuzz-exec-before', 'print', 'emit', 'dump']:
```

Reply:
This is just a substring match on the string containing all the passes, so these are already filtered out by `fuzz`.

Reply:
Oh right, `joined_passes` is a string...